### DISCUSSION TITLE
docker/apptainer

### DISCUSSION THREAD

**Discussion by SSPuliasis** [vdemichev/DiaNN#632] [Q&A]
Hi,

I'd like to run DIA-NN on a HPC with linux but am not authorised to install it. Using apptainer was recommended and I have managed to pull both of the containers on this page (https://hub.docker.com/search?q=dia-nn), but cannot  get any further without encountering the following:

INFO:    underlay of /etc/localtime required more than 50 (68) bind mounts

I'm not very familiar with docker/apptainer, so if anyone has managed to successfully run DIA-NN this way and can provide basic advice or instructions on how to get it running, it would be greatly appreciated

Thanks,
Sophia

**Comment by vdemichev** [comment]
Hi Sophia,

Not sure if this can help, I am not expert on this myself. However the DIA-NN installer for Linux does nothing but just unpacks the DIA-NN files to a particular location. So if you install it on another machine and then just copy paste the files to the target one, this should work, provided the target OS is modern enough (e.g. Ubuntu 18.04 or later).

Best,
Vadim
